File: Radio/app/src/main/java/brokencoin/net/radio/activities/MainActivity.java
package brokencoin.net.radio.activities;

import android.support.v4.app.FragmentManager;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import brokencoin.net.radio.fragments.MainFragment;
import brokencoin.net.radio.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        FragmentManager fm = getSupportFragmentManager();
        MainFragment mainFragment = (MainFragment)fm.findFragmentById(R.id.container_main);

        if (mainFragment == null) {
            mainFragment = MainFragment.newInstance("one", "two");
            fm.beginTransaction().add(R.id.container_main, mainFragment).commit();
        }
    }
}

File: CanaryApps/app/src/main/java/brokencoin/net/canary_testing/MainActivity.kt
package brokencoin.net.canary_testing

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var age : Int? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    fun sum (a: Int, b:Int):Int {
        return a + b
    }

    fun helloWorld(view: View) {

    }

    fun makeThing(view: View){
        //val thingOne = Thing("Thing One", 5, "Do the first thing")

        val name = Name.text.toString()
        if (!Age.text.toString().equals("")) {
            age = Age.text.toString().toInt()
        }
        val job = Job.text.toString()
        val thingOne = Thing(name, age, job)

        textView.text = "Name: " + thingOne.name + "\n" +  "Age: " + thingOne.age + "\n" + "Job: " + thingOne.job

    }
}
